### PR TITLE
Remove unused bindings from asn1.py

### PR DESCRIPTION
**Pull request by davidben**
This also includes a couple removals from x509v3.py which also reference ASN1_ITEM_EXP. Otherwise, I'll probably do these one file at a time barring other preferences from you all.

(Going to let the CI confirm I didn't miss any references.)

### CODE REVIEW

File: src/_cffi_src/openssl/asn1.py [hunk 1]
```diff
@@ -117,39 +84,16 @@
...
 BIGNUM *ASN1_INTEGER_to_BN(ASN1_INTEGER *, BIGNUM *);
 ASN1_INTEGER *BN_to_ASN1_INTEGER(BIGNUM *, ASN1_INTEGER *);
 
-/* These isn't a macro the arg is const on openssl 1.0.2+ */
-int ASN1_GENERALIZEDTIME_check(ASN1_GENERALIZEDTIME *);
-int ASN1_UTCTIME_check(ASN1_UTCTIME *);
-int ASN1_TIME_check(ASN1_TIME *);
-
-/* Not a macro, const on openssl 1.0 */
-int ASN1_STRING_set_default_mask_asc(char *);
```
Comment (alex) [inline src/_cffi_src/openssl/asn1.py]: This is used in pyOpenSSL and thus is needed :-/

**Comment by alex** [comment]
pyOpenSSL appears to also need `MBSTRING_UTF8`

**Comment by alex** [comment]
Thanks @davidben !

**Comment by davidben** [comment]
Oops, thanks for finishing that up. I totally forgot about these two PRs.

**Comment by reaperhulk** [comment]
No worries, thanks for doing the initial work!

**Pull request merged**